dist/images/ovndb-raft-functions.sh

```diff
@@ -272,12 +335,14 @@ ovsdb-raft() {
   echo "=============== ${db}-ovsdb-raft ========== RUNNING"
 
   if [[ "${POD_NAME}" == "ovnkube-db-0" ]]; then
-    # set the election timer value before other servers join the cluster and it can
-    # only be set on the leader so we must do this in ovnkube-db-0 when it is still
-    # a single-node cluster
-    set_election_timer ${db} ${election_timer}
-    if [[ ${db} == "nb" ]]; then
-      set_northd_probe_interval
+    if ${initial_raft_create}; then
+      # set the election timer value before other servers join the cluster and it can
+      # only be set on the leader so we must do this in ovnkube-db-0 when it is still
+      # a single-node cluster
+      set_election_timer ${db} ${election_timer}
+      if [[ ${db} == "nb" ]]; then
+        set_northd_probe_interval
+      fi
     fi
     # set the connection and disable inactivity probe, this deletes the old connection if any
     set_connection ${db} ${port}
```

Comment:
move set_connection into "if ${initial_raft_create}; then" bracket?

Reply:
In the comments it says:
# set_connection() will be called for ovnkube-db-0 pod when :
#    1. it is first started or
#    2. it restarts after the initial start has failed or
#    3. subsequent restarts during the lifetime of the pod

Reply:
yes, but now the condition has changed, if $initial_raft_create is true, it means connection has already been set to the cluster (it passed the db_part_of_cluster() check which does a list-connection).